### ISSUE TITLE
Application Resource Consumption showing resources not in use?

### ISSUE THREAD

**Issue by PlexPro** [eapowertools/qs-telemetry-dashboard#29]
I just installed this today.  I have run the load twice, about fifteen minutes apart.

The app shows data for just one date, which I suppose to be correct.  However, the Application Resource Consumption chart shows resource consumption for eight applications not in use (either archived and not available for anyone to use, or in my workspace and I haven't used them), and only one that is in use.

Is there some sort of app group id being used that is randomly picking up an archive or development version instead of the actual app that has experienced activity?

The same set of nine apps also appear in the "Base RAM Footprint by App" as the only ones out of the list of 186 to show any data.

René V.

**Comment by jparis** [comment]
Hmm, is there a change it's account for some of the monitoring apps that are getting reloaded? Not actually sure what's going on here. It's picking up all this data from the log files, so you can peek into them to see what apps/when it thinks you're accessing them and narrow it down from there?

**Comment by PlexPro** [comment]
Could it have anything to do with the fact that the logs are actually at QlikShare ( C:\QlikShare\ArchivedLogs\vm-win-analytics.polywoodinc.local\Engine\Trace ), and not at C:\ProgramData\Qlik\Sense\Log\Engine\Trace?

I uninstalled, and will reinstall to see if anything be different this time.

**Comment by jparis** [comment]
There should be logs in both places, and it should be pulling logs from both places. There's the current logs in the program data folder, and the archived logs in the archivedlog folder.

**Comment by PlexPro** [comment]
I reinstalled and ran the load tasks without incident.

>> There's the current logs in the program data folder, and the archived logs in the archivedlog folder.

It appears that "C:\ProgramData\Qlik\Sense\Log\Engine\Trace\VM-WIN-ANALYTIC_QixPerformance_Engine.txt" is the only log in the primary location; and the data in that file is what appears in the app.  Nothing from "C:\QlikShare\ArchivedLogs\vm-win-analytics.polywoodinc.local\Engine\Trace" appears in the app.

The "ServerLogFolder" data connection in the "Telemetry Dashboard" app points to "C:\ProgramData\Qlik\Sense\Log\".  This location contains a couple dozen folders, one of which is "Engine", which contains "Trace".  Each place that the variable "vLogRoot" containing the connection name is used, it adds '\Engine\Trace' to the path:  '$(vLogRoot)\Engine\Trace'.

The "ArchivedLogsFolder" data connection points to "\\VM-WIN-ANALYTIC\QlikShare\ArchivedLogs", which is correct; this location contains folder "vm-win-analytics.polywoodinc.local", which then contains seven folders, one of which is "Engine", which contains "Trace".  Each place that the variable "vLogArchive" containing the connection name is used, it adds the computer name to the path -- but not '\Engine\Trace'.

In the calls that use the "vLogArchive" variable, I added 'Engine\Trace\' to see if that would fix it; no good.  Then I tried eliminating the final '\', and that did not work either.

Finally, I changed the connection string of the data connection in the QMC to include the final levels of the path:  "\\VM-WIN-ANALYTIC\QlikShare\ArchivedLogs\vm-win-analytics.polywoodinc.local\Engine\Trace", and removed them along with the variable "vComputerName" from the statements in the load script.  Now I get about ten days worth of data, instead of just a couple of minutes.  Of course, in a multi-node environment that might not work...

It strikes me, I believe that the beginning of the data is the day that I installed this app the first time.  Why isn't it getting data prior to its installation?
